refinery/core/management/commands/activate_user.py

```diff
@@ -9,8 +9,11 @@
 class Command(BaseCommand):
     help = "Activate user account"
 
-    def handle(self, username, **options):
+    def add_arguments(self, parser):
```

Comment:
changes from outparse argparse change. Writing separate issue to deal with other base commands, but changed this one for unit tests to pass.